core/src/main/java/com/mapzen/android/graphics/model/MarkerManager.java

```diff
@@ -1,21 +1,39 @@
 package com.mapzen.android.graphics.model;
 
 import com.mapzen.android.graphics.internal.StyleStringGenerator;
+import com.mapzen.tangram.LngLat;
 import com.mapzen.tangram.MapController;
 import com.mapzen.tangram.Marker;
 
+import android.graphics.drawable.Drawable;
+
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Manages {@link BitmapMarker} instances on the map.
  */
 public class MarkerManager {
```

Comment:
I always get confused with this `Marker` name usage. I know its because multiple reasons including compatibility with how `Marker`->`MapData` usage was and how `BitmapMarker`->`Tangram::Marker` usage is. But can we try to keep the naming such that it distinguishes between sdk markers (tangram mapdata) and sdk bitmapmarkers (tangram markers)?

Reply:
I agree, the naming is not ideal. I'll update to `BitmapMarkerManager` for clarity